### DISCUSSION TITLE
HD300s for Sale - 2 units - $30 each including shipping (Canada only)

### DISCUSSION THREAD

**Discussion by unimpededglare** [OpenSageTV/Community#29] [Marketplace]
I have two fully working HD300s. Each has the original power supply and remote, however I no longer have the breakout cables for either.

I"m selling them for $30 each, including shipping. I'm only selling them within Canada as I don't want to deal with international shipping.

I'm selling them for a low price just to get them gone and it seems like a better alternative than just throwing them in the garbage.

**Comment by bforse** [comment]
Are the HD300s still available?
I will take both for $60 and I am in Canada.

**Reply by unimpededglare** [reply]
Yes. They are both still available. I’m not seeing a way to send private messages on GitHub, I guess email would be the best option to talk. What is your email address?

**Reply by bforse** [reply]
bforse@gmail.com

**Reply by unimpededglare** [reply]
Email sent.

**Comment by unimpededglare** [comment]
Both have been sold.
